bionic/aip/main.py

```diff
@@ -0,0 +1,70 @@
+"""
+This module is run as main in order to execute a task on a worker.
+"""
+
+import logging
+import os
+import sys
+
+from bionic.deps.optdep import import_optional_dependency
+
+
+def run():
+    # Scope the import to this function to avoid raising for anyone not using it.
+    blocks = import_optional_dependency("blocks", raise_on_missing=True)
```

Comment:
WDYT about omitting the `raise_on_missing` parameter in these calls?

Reply:
ah, I didn't realize that the default is `True`. I'll remove it from everywhere in AIP.